BusinessTampereTrafficMonitoring/stream_reader/stream_reader.py

```diff
@@ -0,0 +1,102 @@
+import logging
+import queue
+import time
+from collections import deque
+from threading import Thread
+
+import cv2
+import numpy as np
+
+
+logger = logging.getLogger(__name__)
```

Comment:
I don't think the logger is ever used? Using the logging module is a good idea but the code still seems to use prints instead of the logger.

Reply:
Correct! Modified the code to actually log the errors instead of print them.